auth/api/iam/api.go

```diff
@@ -252,6 +251,24 @@ func (r Wrapper) OAuthClientMetadata(ctx context.Context, request OAuthClientMet
 
 	return OAuthClientMetadata200JSONResponse(clientMetadata(*identity)), nil
 }
+func (r Wrapper) PresentationDefinition(_ context.Context, request PresentationDefinitionRequestObject) (PresentationDefinitionResponseObject, error) {
```

Comment:
Since the s2s-flow is custom, I don't see many other implementations and users of the endpoint. So validation could involve getting the definition again given the scope, and comparing it against the submitted PE?

Reply:
backends do not have to check this, they only need to check if given VCs are enough for the given scope.

Reply:
I just realized that the PD.id generated by the backends will be thrown away when all PDs are merged